policy/modules/system/logging.te

```diff
@@ -607,9 +607,10 @@ files_read_kernel_symbol_table(syslogd_t)
 files_var_lib_filetrans(syslogd_t, syslogd_var_lib_t, { file dir })
 
 fs_getattr_all_fs(syslogd_t)
-fs_read_efivarfs_files(syslogd_t)
 fs_search_auto_mountpoints(syslogd_t)
 fs_search_cgroup_dirs(syslogd_t)
+fs_read_efivarfs_files(syslogd_t)
```

Comment:
Why has this line moved here? It stops following alphabetical order.

Reply:
It's moved together with the other added lines, because make both are required for the same thing. I can move it back if you prefer.

Reply:
Moved  it up now.

Reply:
Thanks. It is not nitpicking, it helps all people going through the rules. It is particularly more important in complex modules.